tests/conftest.py

```diff
@@ -6939,3 +6941,80 @@ def factory(interval):
         )
 
     return factory
+
+
+@pytest.fixture(scope="class")
+def reduce_replication_delay_setup(add_env_vars_to_noobaa_core_class):
+    """
+    A fixture to reduce the replication delay to one minute.
+
+    Args:
+        new_delay_in_miliseconds (function): A function to add env vars to the noobaa-core pod
+
+    """
+    log.warning("Reducing replication delay")
+
+    def factory(new_delay_in_miliseconds=60 * 1000):
+        new_env_var_tuples = [
+            (constants.BUCKET_REPLICATOR_DELAY_PARAM, new_delay_in_miliseconds),
+            (constants.BUCKET_LOG_REPLICATOR_DELAY_PARAM, new_delay_in_miliseconds),
+        ]
+        add_env_vars_to_noobaa_core_class(new_env_var_tuples)
+
+    return factory
+
+
+@pytest.fixture()
+def log_based_replication_setup(
```

Comment:
A more general setup fixture is in the works to also handle the Azure case, so if we're moving this to conftest.py we should make it clear in it's name:
```suggestion
def aws_log_based_replication_setup(
```